README.md

```diff
@@ -1,32 +1,30 @@
-Opt Out
-==============================
+<p align="center"> <img width="250" height="250" src="opt_out_logo.png"> </p>
 
-Our study into online misogyny and how to model it
+<p></p> <h1 align="center"> Opt Out Tools </h1>
+<p></p> <h2 align="center"> Open Source Modeling - Research & Production Code </h2>
 
-Project Organization
+Quick links:
+
+- [Repository Structure](#Repository-Structure)
+- [Purpose of the Repository](#Purpose-of-the-Repository)
+- [How the Repository Works?](#How-the-Repsitory-Works?)
+- [Use the Repository](#Use-the-Repository)
+- [Code of Conduct](#Code-of-Conduct)
+
+
+Repository Structure
 ------------
 
-    ├── LICENSE
-    ├── Makefile           <- Makefile with commands like `make data` or `make train`
-    ├── README.md          <- The top-level README for developers using this project.
-    ├── data
-    │   ├── external       <- Data from third party sources.
-    │   ├── interim        <- Intermediate data that has been transformed.
-    │   ├── processed      <- The final, canonical data sets for modeling.
-    │   └── raw            <- The original, immutable data dump.
-    │
-    ├── docs               <- A default Sphinx project; see sphinx-doc.org for details
-    │
-    ├── models             <- Trained and serialized models, model predictions, or model summaries
-    │
-    ├── notebooks          <- Jupyter notebooks. Naming convention is a number (for ordering),
-    │                         the creator's initials, and a short `-` delimited description, e.g.
-    │                         `1.0-jqp-initial-data-exploration`.
-    │
-    ├── references         <- Data dictionaries, manuals, and all other explanatory materials.
+    ├── CONTRIBUTING.md    <- How to contribute to this repository
+    ├── README.md          <- The top-level desciption for people interested
+    ├── data               <- Our dataset. Access credentials required
+    │    
+    ├── models             <- Trained and serialized models fit for production
     │
+    ├── notebooks          <- Jupyter notebooks. 
+    │    
     ├── reports            <- Generated analysis as HTML, PDF, LaTeX, etc.
-    │   └── figures        <- Generated graphics and figures to be used in reporting
+    │                      <- Eventually will contain dashboard
```

Comment:
Very minor: this is speculative and should not be there. Integrating dashboard within a DS repo is kind of a big task and may belong to an entirely different place. In this sens I would put this e.g. in our project cards.